.github/workflows/workflow.yml

```diff
@@ -0,0 +1,108 @@
+name: fight-pandemics
+
+on: push
+
+jobs:
+  build_images:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        app:
+          - backend
+          - client
+          - geo-service
+      fail-fast: true
+    steps:
+      - uses: actions/checkout@v2
+      - if: matrix.app == 'client'
+        env:
+          AIRTABLE_API_KEY: ${{ secrets.AIRTABLE_API_KEY }}
+          AIRTABLE_BASE: ${{ secrets.AIRTABLE_API_BASE }}
+        run: >
+          echo "REACT_APP_AIRTABLE_API_KEY=${AIRTABLE_API_KEY}" > ./client/.env
+          echo "REACT_APP_AIRTABLE_BASE=${AIRTABLE_BASE}" >> ./client/.env
+          cat .env
+      - name: docker
+        uses: docker/build-push-action@v1.0.1
+        with:
+          path: ./${{ matrix.app }}
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          repository: fightpandemics/${{ matrix.app }}
+          tag_with_ref: true
+  deploy_review:
+    runs-on: ubuntu-latest
+    needs: build_images
+    steps:
+      - uses: actions/checkout@v2
+      - id: get-branch-name
+        uses: ypicard/get-branch-name-github-action@v1
+      - env:
+          AUTH_SECRET_KEY: ${{ secrets.AUTH_SECRET_KEY }}
+          AUTH_CLIENT_ID: ${{ secrets.AUTH_CLIENT_ID }}
+          AWS_DEFAULT_REGION: us-east-1
+          AWS_ACCESS_KEY_ID: ${{ secrets.REVIEW_AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.REVIEW_AWS_SECRET_ACCESS_KEY }}
+        run: |
+          cat << EOF > main.tf
+          terraform {
+            backend "s3" {
+              bucket = "fp-review-terraform-state"
+              region = "us-east-1"
+              key = "${{ steps.get-branch-name.outputs.branch }}.tfstate"
+            }
+          }
+          module "main" {
+            source = "github.com/FightPandemics/tf-fargate-task//module"
+            image_tag = "${{ steps.get-branch-name.outputs.branch }}"
+            fp_context = "review"
+            subdomain = "${{ steps.get-branch-name.outputs.branch }}"
+            backend_env_variables = [
+              {
+                name = "PORT"
+                value = "8000"
+              },
+              {
+                name = "MONGO_URI"
+                value = "localhost:27017"
```

Comment:
I think it's best if this includes a database name since we now use `Mongoose.createConnection()` and that can connect to a specific database, or maybe we need to add a separate setting, I'm currently using `127.0.0.1:27017/fightpandemics` locally but I guess we will move this to AWS DocumentDB or Mongo Atlas?